app/controllers/users_controller.rb

```diff
@@ -1,3 +1,24 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = t("users.new.success")
+      redirect_to @user
+    else
+      render "new", status: :unprocessable_entity
```

Comment:
```suggestion
      render :new, status: :unprocessable_entity
```
thêm 1 flash báo lỗi nữa e nhé